phase02-code review/src/main/java/Processor.java

```diff
@@ -3,31 +3,31 @@
 public class Processor {
     private static Processor instance;
     private final FileReader fileReader;
-    private final InvertedIndex invertedIndex;
-    private final View view;
+    private final HashInvertedIndex hashInvertedIndex;
+    private final ConsoleView consoleView;
     private ArrayList<String> withoutSignDocs;
     private ArrayList<String> plusDocs;
     private ArrayList<String> minusDocs;
 
     public Processor() {
         fileReader = FileReader.getInstance();
-        invertedIndex = InvertedIndex.getInstance();
-        view = View.getInstance();
+        hashInvertedIndex = HashInvertedIndex.getInstance();
+        consoleView = ConsoleView.getInstance();
         withoutSignDocs = new ArrayList<>();
         plusDocs = new ArrayList<>();
         minusDocs = new ArrayList<>();
     }
 
     private ArrayList<String> getWithoutSignDocs() {
```

Comment:
leave a comment for algorithm of this function

Reply:
the algorithm of function is so simple if you look at the name of it.